src/Agent.Worker/Build/TeeTFCommandManager.cs

```diff
@@ -47,14 +48,14 @@ public sealed class TeeTFCommandManager : AgentService, ITeeTFCommandManager
         {
             ArgUtil.NotNullOrEmpty(version, nameof(version));
             ArgUtil.NotNullOrEmpty(directory, nameof(directory));
-            await RunCommandAsync("get", $"-version:{version}", "-recursive", "-overwrite", directory);
+            await RunCommandAsync(FormatFlags.None, "get", $"-version:{version}", "-recursive", "-overwrite", directory);
         }
 
         public string ResolvePath(string workspace, string serverPath)
         {
             ArgUtil.NotNullOrEmpty(workspace, nameof(workspace));
             ArgUtil.NotNullOrEmpty(serverPath, nameof(serverPath));
-            string localPath = RunPorcelainCommandAsync("resolvePath", $"-workspace:{workspace}", serverPath).GetAwaiter().GetResult();
+            string localPath = RunPorcelainCommandAsync(FormatFlags.None, "resolvePath", $"-workspace:{workspace}", serverPath).GetAwaiter().GetResult();
```

Comment:
Yea I'm not sure either. During one point I had it async/await all the way through and passing cancellation token as well. The change rippled all the way through to taskrunner, and seemed like overkill. Stepping back now and thinking about it again, either way seems fine. I even leaning toward an async interface now... We can discuss further, I don't want it to hold up this PR since it's unrelated.